api_grpc/worker.go

```diff
@@ -447,9 +449,15 @@ func notifyWorkersPremieres() {
 		}
 		workerIndex := getWorkerWithLeastWorkload(workers)
 		workers[workerIndex].Workload += 3
+		ingestServer, err := dao.GetBestIngestServer()
+		if err != nil {
+			log.WithError(err).Error("Can't find ingest server")
```

Comment:
Maybe we should stop here when an error happens?

Reply:
You're right, we absolutely should. Pushed a fix